### PR TITLE
Fix/netlify redirects

### PR DESCRIPTION
**Pull request by DivyanshBatham**
Added netlify.toml file

Fixes Issue #25

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[build]
```
Comment (Arjun-sna) [inline netlify.toml]: I think configuration for `build` is not required.

Comment (DivyanshBatham) [reply netlify.toml]: I myself am not sure about this, doing it for the first time.

Comment (DivyanshBatham) [reply netlify.toml]: I'll remove the `[build]` part, and sent another commit then?

Comment (DivyanshBatham) [reply netlify.toml]: @Arjun-sna Added another commit, removing `[[build]]` part.

**Pull request merged**